ui/tests/e2e/extensions.spec.js

```diff
@@ -0,0 +1,133 @@
+import { expect, test } from '@playwright/test';
+import { ENV } from './env';
+
+// Extensions Section Tests
+test.describe('Extensions Section Tests', () => {
+  test.beforeEach(async ({ page }) => {
+    // Visit The Meshery Serve URL
+    await page.goto(ENV.MESHERY_SERVER_URL);
+
+    const extensionsNav = page.locator('[data-test="navigation"] >> text=Extensions');
+    await extensionsNav.click();
+  });
+
+  test('Verify Kanvas Snapshot using data-testid', async ({ page }) => {
+    // Verify heading using data-testid
+    await expect(page.getByTestId('kanvas-snapshot-heading')).toBeVisible();
+
+    // Verify description using data-testid
+    await expect(page.getByTestId('kanvas-snapshot-description')).toBeVisible();
+
+    // Verify enable button using data-testid
+    const enableButton = page.getByTestId('kanvas-snapshot-enable-btn');
+    await expect(enableButton).toBeVisible();
+    await expect(enableButton).toBeDisabled();
+
+    // Verify snapshot logo using data-testid
+    await expect(page.getByTestId('kanvas-snapshot-image')).toBeVisible();
+  });
+
+  test('Verify Performance Analysis Details', async ({ page }) => {
+    // Verify heading using data-testid
+    await expect(page.getByTestId('performance-analysis-heading')).toBeVisible();
+
+    // Verify enable button using data-testid
+    const performanceEnableButton = page.getByTestId('performance-analysis-enable-btn');
+    await expect(performanceEnableButton).toBeVisible();
+    await expect(performanceEnableButton).toBeDisabled();
+  });
+
+  test('Verify Kanvas Details', async ({ page, context }) => {
+    // Verify Kanvas heading using data-testid
```

Comment:
Can we get rid of these comments the code here is very clear and some of the other places too. Thank you